### ISSUE TITLE
[MU4 Issue] VST Effects - Crash on Window Close

### ISSUE THREAD

**Issue by lengthwave** [musescore/MuseScore#11683]
**Describe the bug**
When closing the VST effects window, MuseScore crashes (sometimes crashes immediately, sometimes after a few times open/closing window).

**To Reproduce**
1. Go to Mixer
2. Select a track and click on Audio FX
3. VST Effects window will open
4. Close VST Effects window
5. See error (maybe, it is not consistent)

**Expected behavior**
No crashie. 

**Screenshots**

https://user-images.githubusercontent.com/30370609/169241264-bb908322-e4f6-4f5b-8d41-4fefe98a5a5d.mov

**Platform information**
OS: macOS 12.3
Arch.: x86_64
MuseScore version (64-bit): 4.0.0-2349650330
revision: github-musescore-musescore-8c123de

**Comment by lengthwave** [comment]
FYI @Tantacrul

**Comment by lengthwave** [comment]
I'll try to reproduce with free VST.

**Comment by Tantacrul** [comment]
I wonder if this issue is still happening? I have not experienced this myself in months.

**Comment by DmitryArefiev** [comment]
@Tantacrul This **still happens** with **Arturia Rev PLATE-140** (Mac12.5, latest master). 

But no crash with Valhalla Supermassive, Valhalla Vintage Verb, NI Passive EQ

**Comment by Tantacrul** [comment]
Moving to post beta because many of the crashes we were experiencing are fixed and this is looking like a plugin-specific issue now.

We're going to have a lot of these over the coming months. We should focus on core playback for now.

**Comment by DmitryArefiev** [comment]
Now it's much better after #13848, it's harder to crash MU

And another issue with Arturia Rev PLATE-140 is changing settings doesn't apply immediately, you should reopen score

https://user-images.githubusercontent.com/90187801/197514653-278d23fb-26b0-4a5a-aaa6-fc2e7838eb67.mp4

**Comment by davebtc67** [comment]
Garritan still crashing MU4

**Comment by Tantacrul** [comment]
> Garritan still crashing MU4

Unfortunately, we can't fix multiple issues in one thread. It would be great if you'd log this separately.

**Comment by cbjeukendrup** [comment]
> Garritan still crashing MU4

Already reported in #13867

**Comment by CubikingChill** [comment]
@DmitryArefiev Would you be able to briefly re-test if this issue still exist? To my knowledge there has been some substantial improvements in VST handling.

**Comment by DmitryArefiev** [comment]
Tested in **4.7.4** on **Win10/Mac13.7.8**. **Crash no longer occurs**

**Changing effect settings not applied in real time** - this still happens, but I'll add this to #33124 since this related to Arturia plugins
